### ISSUE TITLE
Regular emoji throwing spellcheck warnings

### ISSUE THREAD

**Issue by ryanblock** [tbroadley/spellchecker-cli#41]
This library is great!

Noticed that regular emoji are throwing warnings. For example:
```
./src/shared/en/aws/guides-custom-dns.md
     63:9-63:10  warning  `️` is misspelt  retext-spell  retext-spell
      80:4-80:5  warning  `️` is misspelt  retext-spell  retext-spell
  107:42-107:43  warning  `️` is misspelt  retext-spell  retext-spell
```
The lines in question:

63: `> 🤷🏽‍♀️ DNS propagation can take time: have patience!`
80: `> ⛳️ Tip: These instructions will serve your app's production environment...`
107: `...grab a cup of coffee or tea ☕️ – it can take a few minutes while... `

May take a crack at a fix (would first try implementing [`remark-emoji`](https://github.com/rhysd/remark-emoji), would that kind of thing be accepted as a PR?

**Comment by tbroadley** [comment]
Good find, thanks! I'd absolutely take a PR to fix this.

**Comment by ryanblock** [comment]
This turned out to be a bit more subtle than I'd expected / hoped. There's something somewhere triggering failures on standard unicode emoji variation selector char (invisible, UTF-8 hex: `0xEF 0xB8 0x8F` [`efb88f`]).

Haven't yet had the time to track it down beyond that, but for reference I've attached a single line array containing the char in question, which does successfully suppress the erroneous warnings: [dictionary.js.zip](https://github.com/tbroadley/spellchecker-cli/files/2508348/dictionary.js.zip)

P.S. I super love this project. Really nice work. It's now a mandatory build step for all our docs sites! 💕

**Comment by j-f1** [comment]
Looks like you’ve got [Variation Selector 16](https://www.fileformat.info/info/unicode/char/fe0f/index.htm), which coerces characters into emoji.

**Comment by tbroadley** [comment]
#43 should fix this issue by filtering out Variation Selectors 15 and 16 before spellchecking.

**Comment by ryanblock** [comment]
Kudos for getting to this! It has been on my to-do list. 🌈

I'll go give it a peek!

**Comment by ryanblock** [comment]
Issue no longer replicates locally! (Also tested with skin tone, family, and some other variations.) Woo!

**Comment by tbroadley** [comment]
Great! I just released v4.0.1, which contains the fix.
